Makefile.core.mk

```diff
@@ -240,7 +240,7 @@ undefine BUILDX
 endif
 
 .PHONY: docker-buildx
-docker-buildx: test build-all ## Build and push docker image with cross-platform support.
+docker-buildx: build-all ## Build and push docker image with cross-platform support.
```

Comment:
Should we test somewhere else? Before publishing the bundle maybe?

Reply:
all merged code is already tested. why test again?